docs/dev/README.md

```diff
@@ -219,7 +223,7 @@ options, e.g.:
 
 ## Debugging Tips
 An easy way to debug the plugin is to run in
-[Spark local mode](https://jaceklaskowski.gitbooks.io/mastering-apache-spark/local/spark-local.html).
+[Spark local mode](https://spark.apache.org/docs/latest/spark-standalone.html).
```

Comment:
This section is referring to running Spark in local mode, but we're changing the link to refer to Spark standalone mode which is not the same thing.

Reply:
Updated link to http://spark.apache.org/docs/latest/submitting-applications.html#master-urls